### PR TITLE
netobserv: Turn on "ignore_review_state"

### PR DESCRIPTION
**Pull request by jotak**
Prefer an explicit approval to trigger the merge, that can be different
from the lgtm/code review. Avoids accidental merge or necessity to /hold
preemptively

### CODE REVIEW

**Comment by jotak** [comment]
looks like this isn't supported
`make prow-config` would cancel my changes :-/

**Comment by jotak** [comment]
`ignore_review_state` should make the `approved` label NOT set after code review approval in github UI.
Since we also have `review_acts_as_lgtm: true`, it means that the code review from github UI will set `lgtm` and not `approved`, which is what we want

**Review by mariomac — APPROVED** [review]

**Pull request merged**
